requirements.txt

```diff
@@ -17,5 +17,6 @@ rfc3987==1.3.8
 GitPython==3.1.3
 hvac==0.10.4
 docker==4.2.1
+py7zr==0.7.4
```

Comment:
is supporting 7zip really necessary?

Reply:
We can definitely remove it and document the supported archiving/compression formats.

Reply:
It might be preferred to start without it to keep our dependencies low and if users ask for it explicitly we can easily re-add it. :)